backend/internal/adapters/primary/handlers/internal/gemservice.go

```diff
@@ -25,14 +26,27 @@ func NewGemService(repository gems.Repository) *gemService {
 }
 
 func (s *gemService) Drop(c context.Context, r *gemsapi.DropRequest) (*gemsapi.DropResponse, error) {
-	gem, err := s.usecases.Drop(r.Message)
+	gemMessage := r.GetGemMessage()
+	gem := gems.Gem{
+		// TODO: generate ID here, not sure how.
+		ID:         123,
```

Comment:
Not 100% certain, but is it necessary to provide all the fields? I think we can probably omit the ID. Likewise for created_at since these will be populated by the db

Reply:
removed